public/class-admwswp-public.php

```diff
@@ -202,6 +203,18 @@ public static function weblinkWidget($attr)
                 $webLinkArgs['eventListOrder']['field'] = $event_list_order_field;
                 $webLinkArgs['eventListOrder']['direction'] = $event_list_order_direction;
 
+                if ($event_list_order) {
+                    $webLinkArgs['eventListOrder'] = array();
+                    $event_list_order = explode(",", $event_list_order);
+                    foreach ($event_list_order as $orderBy) {
+                        $order = explode("-", $orderBy);
+                        $webLinkArgs['eventListOrder'][] = array(
+                            'field' => $order[0],
+                            'direction' => $order[1],
```

Comment:
do we want to trim the values from trailing spaces ? as an addition space will cause graphql errors

Reply:
yes we can do that